src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -385,7 +386,7 @@ export default defineComponent({
      * Sets default size if not provided.
      */
     const _size = computed(() => {
-      if (isBoxed && !props.size) {
+      if (isBoxed.value && !props.size) {
```

Comment:
Aargh! I wonder if there's a linter to check for `.value` not being added.

Reply:
I'm looking forward to the [reactivity transform](https://vuejs.org/guide/extras/reactivity-transform.html) proposal that enables the compiler to automatically handle this for us.

Reply:
The TypeScript ESLint plugin has type-aware linting that is able to catch things like this as well.